web/account.go

```diff
@@ -331,6 +331,13 @@ func CreateKey(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Catch cases where key is already registered
+	_, ok = data.GetSSHKey(key.Fingerprint)
+	if ok {
+		PrintErrorJSON(w,r,"This key is already registered", http.StatusConflict)
```

Comment:
Missing spaces between args as well.
Running a `gofmt` would just take care of it automatically.